lib/utils/schedulingUtility.dart

```diff
@@ -31,12 +41,22 @@ class SchedulingUtility {
     }
   }
 
-  static bool isSchedulingConflict(DayTime dayTimeOne, DayTime dayTimeTwo,
-      Campus campusOne, Campus campusTwo) {
-    return isCloseTime(dayTimeOne, dayTimeTwo, timeRequired(campusOne, campusTwo));
+  static bool isSchedulingConflict(Lecture lectureOne, Lecture lectureTwo) {
+    return isCloseTime(lectureOne, lectureTwo, timeRequired(lectureOne.campus,
+        lectureTwo.campus));
   }
 
   static String constructSchedulingConflictText(Lecture one, Lecture two) {
+    int startOne = convertToMinutes(one.startDayTime.hour,
+        one.startDayTime.minute);
+    int endOne = convertToMinutes(one.endDayTime.hour,
+        one.endDayTime.minute);
+    int startTwo = convertToMinutes(two.startDayTime.hour,
+        two.startDayTime.minute);
+    if (startTwo >= startOne && startTwo <= endOne) {
+      return "Please consider that the schedules between " + one.course.name +
```

Comment:
Good one, I just wanted to open an issue for this... Great if you already solved it.